pkg/kubelet/volume_host.go

```diff
@@ -76,6 +80,10 @@ func (kvh *kubeletVolumeHost) GetKubeClient() clientset.Interface {
 	return kvh.kubelet.kubeClient
 }
 
+func (kvh *kubeletVolumeHost) GetSecretFunc() func(namespace, name string) (*v1.Secret, error) {
```

Comment:
I'd like to avoid adding any new methods to the `kubeletVolumeHost`. As you can tell by having it return `nil` for the controllers, it was a poor abstraction that we need to refactor. That said, since the new way of passing parameters from the volume manager to volume plugins doesn't exist yet, it's fine to add the method here, and when we do the refactor we'll clean this up.